tests/features/joinup_federation/import.feature

```diff
@@ -213,8 +225,7 @@ Feature: As a site moderator I am able to import RDF files.
     And I delete the provenance activity of "http://example.com/owner/1" entity
     And I delete the provenance activity of "http://example.com/owner/2" entity
     And I delete the provenance activity of "http://example.com/contact/1" entity
-    And I delete the provenance activity of "http://example.com/license/1" entity
-    And I delete the "A federated licence" licence
+    And I delete the "Federated open license" licence
```

Comment:
This is not needed. The licence has been created and is tracked by Behat. So, it will be removed automatically.